.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,16 +8,25 @@ Please label this PR with one of the following labels, depending on the scope of
 - Docs
 -->
 
-## What does this PR do?
+## Proposed commit message
 
 <!-- Mandatory
-Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
--->
+Explain here the changes you made on the PR.
```

Comment:
I would remove this one as we already have "Please explain:" just after.
Otherwise everything is good to me!

Reply:
I guess this is a matter of style. The text here reflects the text that was previously there, just now on two lines. The thinking that I had with retaining the text as it is was that the first line is the overall goal of the section, while the second is the specific parts that should be addressed. If you still want me to remove it given that, I'm happy to.

Reply:
I don't have a problem with it as it is, but if you can make it shorter people are more likely to read it :p

Reply:
I agree about that. We need to have reviewers on board to point to this, so the main issue is whether that will happen.